### ISSUE TITLE
Lack of proper UAssetAPI licensing

### ISSUE THREAD

**Issue by atenfyr** [SolicenTEAM/KismetEditor#3]
This project redistributes a compiled DLL of [UAssetAPI](https://github.com/atenfyr/UAssetAPI), which is licensed under the MIT license. As such, the [UAssetAPI license text](https://github.com/atenfyr/UAssetAPI/blob/master/LICENSE) must be included somewhere within the repository and with any binary distribution of the software in order to comply with the intellectual property rights that are reserved by the original contributors of UAssetAPI according to United States law.

**Comment by DenisSolicen** [comment]
Hi, I'm a little sorry for the delay in responding, I don't check `Issues` very often, I'm glad that you turned your attention to this project. According to your message, I have added the license in accordance with the necessary standards, I hope there will be no more problems.

And thank you a lot for UAssetAPI, it helps us translators a lot in various cases!
Once again, I apologize for the delay, all the best.

**Comment by DenisSolicen** [comment]
@atenfyr if you have more suggestions on license subjects and suggestions on how best to include your licensing in the repository, you could just **reopen this issues**, and don't write this DMCA request.

If you remain silent for the next day and the result is zero, I will appeal your request as unfounded.

**Comment by atenfyr** [comment]
Thanks for reaching out. Your changes to the repository are more than satisfactory - the DMCA request was filed several weeks before your initial response, and was processed by GitHub despite you having made the necessary changes to the repository between the time it was filed and the time that GitHub processed the request. Prior to the time I assume that you received contact from GitHub, I had already contacted GitHub to inform them that the issue was resolved. The fact that you received further communication about this matter seems to be an error on GitHub's part, and I will pursue no further action against you on this matter. Apologies for the inconvenience and the miscommunication.
